lib/ProductOpener/Display.pm

```diff
@@ -6256,6 +6256,9 @@ HTML
 	my $status = $request_ref->{status};
 	if (defined $status) {
 		print header ( -status => $status );
```

Comment:
There's already a print header that has been applied in the if elsif else loop above, that seems strange. Am I missing something?

Reply:
Those only set cookies or cache policies. The status header is a separate header line, so it should not be (and doesn't appear to be) a problem to sends multiple `print header(...);` calls.

Reply:
It generates an extra empty line, but it seems mod_perl parses the headers anyway, so I guess it's fine.

perl test_header.pl
Expires: Wed, 29 May 2019 09:25:24 GMT
Date: Thu, 30 May 2019 09:25:24 GMT
Content-Type: text/html; charset=UTF-8

Status: 200
Content-Type: text/html; charset=UTF-8